.github/workflows/ci.yml

```diff
@@ -34,6 +34,8 @@ jobs:
         uses: actions/checkout@v4
       - name: Set up tree-sitter
         uses: tree-sitter/setup-action/cli@v1
+        with:
+          tree-sitter-ref: v0.25.6
```

Comment:
Is this pinning the version of `tree-sitter/setup-action/cli@v1` used or our CLI/lib dependency? And if it's the latter should we just replace the relevant line in package.json with `==0.25.6` instead of `^0.25.6` since I assume the action is node based?